examples/hf/t5/pippy_t5.py

```diff
@@ -133,7 +134,16 @@ def run_master(pp_ranks, args):
 
     number_of_workers = len(pp_ranks) - pippy.utils.exclude_master
     print(f"number_of_workers = {number_of_workers}")
-    add_split_points(t5, number_of_workers)
+    # Specify auto_split policy for use by `from_tracing` call later
+    if args.auto_split == "threshold":
+        split_policy = pippy.ModelSplit.split_on_size_threshold(490 * 1e6)
```

Comment:
How do you decide on 490 * 1e6? should this be conditioned on finding the size of the module?

Reply:
Sorry, missed this comment. 
This argument is per user's decision. I picked 490 because it would happen to split the model into 8 stages, connecting well with the rest of the program.